jsonQueries/pythonUDF/udfAgg.json

```diff
@@ -0,0 +1,120 @@
+{
+  "rawQuery": "Stateful agg",
+  "fragments": [
+    {
+      "operators":[
+         {
+            "opName":"MyriaShuffleConsumer",
+            "opType":"ShuffleConsumer",
+            "argOperatorId":2,
+            "opId":3
+         },
+         {
+            "opType":"MultiGroupByAggregate",
+            "argGroupFields":[1,2],
+            "aggregators":[
+               {
+                  "initializers":[
+
+                       {
+                          "rootExpressionOperator":{
+                             "value":"null",
+                             "valueType":"BLOB_TYPE",
+                             "type":"CONSTANT"
+                          },
+                          "outputName":"tm"
+                       }
+                  ],
+                  "updaters":[
+                     {
+                     "rootExpressionOperator":{
+                       "children": [
+                         {
+                           "columnIdx":0,
+                          "type":"STATE"
+                         },
+                         {
+                           "type": "VARIABLE",
+                           "columnIdx": 0
+                         },
+                         {
+                           "type": "VARIABLE",
+                           "columnIdx": 2
+                         },
+                         {
+                           "columnIdx":3,
+                          "type":"VARIABLE"
+                         }
+                       ],
+                       "type": "PYUDF",
+                       "outputType":"BLOB_TYPE",
+                       "name": "udfAgg"
+
+                     },
+                     "outputName":"tm"
+                  }
+                  ],
+                  "emitters":[
+                     {
+                    "rootExpressionOperator":{
+
+                       "columnIdx": 0,
+                       "type": "STATE"
+                       },
+                       "outputName":"voxel"
+
+                     }
+                  ],
+                  "type":"UserDefined"
+               }
+            ],
+            "argChild":3,
+            "opId":4,
+            "opName":"MyriaGroupBy(a; UDA(_count__147))"
+         },
+
```

Comment:
I'm nit-picking, but the indents of this JSON (and a few others) need to be fixed. Also there are unnecessary empty lines.